select.scss

```diff
@@ -0,0 +1,127 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+
+// Whether to draw the default down-arrow icon.
+$select__draw-caret: true !default;
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/triangle';
+@import 'level/level';
+
+
+// Root
+// ---
+
+.c-select {
+    position: relative; // Allow positioning of select element and caret
+
+    display: block;
+    box-sizing: border-box;
```

Comment:
Oh, re: `box-sizing`, yes. I’ve been unsure about whether to add it there. My current thinking is:

- border-box isn’t strictly part of normalizing (content-box is still the cross-browser default)
- components should be as decoupled as possible; it’s a concession to even have them depend on normalization (but one that makes sense)

There’s no reason we can’t describe level as normalize + a few niceties and add border-box there. It’s probably pragmatic as well, since bugs could crop up if we have to remember border-box on every component, as you say.